### ISSUE TITLE
Deprecated Event: currentDocumentChange

### ISSUE THREAD

**Issue by coreybutler** [wakatime/brackets-wakatime#2]
I'm using Bracets 1.1 (latest w/ Extract Preview) and noticed the following warning in the Developer tools:

```
Registering for deprecated event 'currentDocumentChange'. Instead, use MainViewManager.currentFileChange. Error
    at Object.on (/utils/EventDispatcher.js:100:43)
    at Object.<anonymous> (file:///C:/Users/Corey/AppData/Roaming/Brackets/extensions/user/brackets-wakatime/main.js:87:24)
    at Object.context.execCb (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1658:33)
    at Object.Module.check (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:869:55)
    at Object.Module.enable (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1151:22)
    at Object.Module.init (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:782:26)
    at callGetModule (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1178:63)
    at Object.context.completeLoad (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1552:21)
    at HTMLScriptElement.context.onScriptLoad (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1679:29)
```

**Comment by alanhamlett** [comment]
Fixed in v1.0.4 with c60e86db6374b22356f4e080c0b317f894847266

**Comment by alanhamlett** [comment]
Thanks @coreybutler
